toolbox/cluster/set_scale.sh

```diff
@@ -0,0 +1,19 @@
+#! /bin/bash -e
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+source ${THIS_DIR}/../_common.sh
+
+if [ -z "${2-}" ]; then
+    echo "USAGE: ${0} <instance_type> <scale>"
+    exit 1
```

Comment:
Could you add more information about the usage?
in particular, could you mention here some the instance-types that offer GPUs?

and I was wondering how much of it is AWS specific? I was wondering if the name should be `set_scale_aws.sh` or not

Reply:
The `.sh` file itself can forever be agnostic I believe, all platforms just need an instance type name and a number to scale.

The playbooks however might already be aws-specific, not sure. Since we don't have backwards compatibility concerns, I believe it will be easy to generalize them once we want to, so no need to worry about it for now.